### DISCUSSION TITLE
Copy link as Chinese characters instead of URL encode

### DISCUSSION THREAD

**Discussion by People-11** [spencerwooo/onedrive-vercel-index#369] [Ideas]
Copy as https://people11.vercel.app/动漫与影视
not https://people11.vercel.app/%E5%8A%A8%E6%BC%AB%E4%B8%8E%E5%BD%B1%E8%A7%86

**Comment by myl7** [comment]
根据标准（RFC3986），前者其实并不是合法的 URL，但是 browser 能够自动 urlencode 从而转化为正确的 URL。甚至可以直接在地址栏里输入空格，browser 都能自动 urlencode 成 %20。

所以要让 URL“readable” 其实是要让 URL 可以被当前应用当成 URL，比如遵守 [GFM 的 autolink](https://github.github.com/gfm/#autolinks-extension-)。这个各大应用应该没有一个很统一的标准，也没有（应该没有）相关 RFC。

不过或许可以把一部分显然可行的字符，比如 CJK、emoji 等，单独拉出来 urldecode。

**Comment by myl7** [comment]
Ref: #371
